### PR TITLE
[14.0][MIG] stock_picking_report_valued: Migration to 14.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT26456

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
Tested on runbot and code review :+1:

File: stock_picking_report_valued/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+To get the stock picking valued report:
+
+#. Create a Sale Order with stockable products a *Valued picking* able
```
Comment (Tardo) [inline stock_picking_report_valued/readme/USAGE.rst]:
```suggestion
#. Create a Sales Order with storable products a *Valued picking* able
```

**Review by Tardo — APPROVED** [review]
LGTM :+1: Tested on runbot. Only a non-blocking comment.

Maybe the last column (Quantity) can be aligned to right ?¿?

**Comment by pedrobaeza** [comment]
@ernestotejeda please apply suggestion and tell me back.

**Pull request merged**